localstack/services/apigateway/helpers.py

```diff
@@ -387,7 +392,8 @@ def add_base_path_mapping(path, data):
 
     domain_name = get_domain_from_path(path)
     # Note: "(none)" is a special value in API GW:
-    # https://docs.aws.amazon.com/apigateway/api-reference/link-relation/basepathmapping-by-base-path
+    # https://docs.aws.amazon.com/apigateway/api-reference/link-relation/basepathmapping-by-base
```

Comment:
@calvernaz Looks like you may have a certain config in your IDE that strips off lines after 96 (?) characters. Or is this coming from a recent upgrade in `black8` when running `make format`?

This leads to some line hard wraps in this file - can we revert these changes, or alternatively integrate the rules such that they get applied on `make format` as well?

Reply:
@whummer Not on the IDE, this is the result of having the git hooks generated from the `.pre-commit-config.yaml` file. Those hooks use the pyproject.toml configuration and as far as I can see the rule is there.

```
[tool.black]
line_length = 100
include = '(localstack/.*\.py$|tests/.*\.py$)'
extend_exclude = '(localstack/infra|localstack/node_modules)'
```

Reply:
It looks awful I agree, maybe I can normalise the same line length in all the configs.